fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/NodePropertiesTools.java

```diff
@@ -203,9 +203,18 @@ public void removeNodeProperty(final Node node, final String propertyName, final
             if (property.isMultiple()) {
                 final AtomicBoolean remove = new AtomicBoolean();
                 final Value[] newValues = stream(node.getProperty(propertyName).getValues()).filter(v -> {
-                    if (v.equals(valueToRemove)) {
-                        remove.set(true);
-                        return false;
+                    String strVal = "";
+                    try {
+                        strVal = v.getString();
+
+                        if (strVal.equals(valueToRemove.getString())) {
+                            remove.set(true);
+                            return false;
+                        }
+
+                    } catch (RepositoryException e) {
```

Comment:
I've been revisiting my logic and want to make sure I understand the expected behavior.  Here are some cases: 

1) user requests remove of property value
      a) value doesn't exist   (silent completion?)  - ie user requests removal of "foo" when there is no property value "foo"
              Do we tell the user it didn't actually delete anything?? Doesn't seem necessary.
      b) value does exist and is removed successfully
      c) value does exist and is not removed
              create RepositoryException and notify user that remove failed. 

This sounds okay? Am I missing anything?